manage/installing/installation.rst

```diff
@@ -125,11 +125,11 @@ Download the latest Plone unified installer
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
 Download from `the plone.org download page <https://plone.org/download>`_ to your server using wget command. Curl also works.
```

Comment:
It should be easy to extract from the PyPI information we can get as json from https://pypi.python.org/pypi/Products.CMFPlone/json
So we just need a view using `requests`, some caching + some logic (we have to define first) and some html. This is imo beginner level Plone coding.